src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -239,6 +240,7 @@ protected function createConfigurationDefinition()
                     'const' => self::SPACING_ONE,
                     'method' => self::SPACING_ONE,
                     'property' => self::SPACING_ONE,
+                    'trait_import' => self::SPACING_ONE,
```

Comment:
@keradus @SpacePossum isn't this BC break? Now, after update to 3.0.1 everyone get the trait imports changed?

Reply:
I see that the initial decision was it's bugfix and going to 2.19.x

and yeah, each bugfix will change things for folks, thus can be considered BC breaker

Reply:
Was it really a bugfix? I'ts not a problem to you or me as we know exactly what is going on in repo.

I'm thinking about people not familiar that well with the repo and seeing 3.0.1 patch version and wants the bugfixes.

Reply:
No, it was intended for the rule to function this way but was never made to do so. If PHP adds another type of class element definition in the future it will be added to the list as well.